### DISCUSSION TITLE
Will Apps Get Rebuilt When Local Package is Updated

### DISCUSSION THREAD

**Discussion by jrock2004** [vercel/turborepo#7708] [Help] [answered]
### Summary

I want to understand if my apps that rely on a package in the `packages` directory changes to get newest change

### Additional information

So in my packages directory I have a package that is some shared utilities. When I make a change to that will I need to manually rebuild all the apps that rely on it or will it do it for me? Let me know if there is any info you need

**Comment by mehulkar** [comment]
This depends on your app, how it imports packages and how it is configured. Generally speaking turbo does not handle this. However, we _are_ working on a [Watch mode](https://github.com/vercel/turbo/issues/986) feature that may work for you. cc @NicholasLYang

**Reply by jrock2004** [reply]
So in the page we import like `import { init, setEnviromentVariables, ACCESS_BASED } from 'auth-utils';`

In the package json its like `"auth-utils": "workspace:*",`

**Comment by anthonyshew** [comment, marked as answer]
We've got some updated guidance here: https://turbo.build/repo/docs/crafting-your-repository/developing-applications
